circle.yml

```diff
@@ -85,7 +75,7 @@ jobs:
 
     # Accept android licenses
     - run: mkdir -p /usr/local/android-sdk-linux/licenses
-    - run: if [[ ! -e "$ANDROID_HOME/ndk-bundle" ]]; then wget -nv http://dl.google.com/android/repository/android-ndk-r11c-linux-x86_64.zip -O android-ndk-r11c-linux-x86_64.zip && unzip -o android-ndk-r11c-linux-x86_64.zip -d "$ANDROID_HOME" && mv "$ANDROID_HOME/android-ndk-r11c" "$ANDROID_HOME/ndk-bundle"; fi
+    - run: if [[ ! -e "$ANDROID_HOME/ndk-bundle" ]]; then wget -nv http://dl.google.com/android/repository/android-ndk-r11c-linux-x86_64.zip -O $HOME/downloads/android-ndk-r11c-linux-x86_64.zip && unzip -o $HOME/downloads/android-ndk-r11c-linux-x86_64.zip -d "$ANDROID_HOME" && mv "$ANDROID_HOME/android-ndk-r11c" "$ANDROID_HOME/ndk-bundle"; fi
```

Comment:
downloading into shared was causing problems during the checkout phase, since it balks if it's nonempty. Probably the flakes that @chrisnojima was seeing earlier.